CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v51.2.0...main)
 
+* General
+  * Update URL metric character length to 8192 ([#2199](https://github.com/mozilla/glean/pull/2199))
```

Comment:
and truncate!

Reply:
Fixed in https://github.com/mozilla/glean/pull/2199/commits/591e57f6bb01c435d1e08e5291d78bcdfd44f815